Anlab.Jobs.MoneyMovement/appsettings.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "MoneyMovement": {

```

Comment:
@jSylvestre I would generalize as "Financial" so we can use along with other financial vars and secure them all the same way